apps/vaults/hooks/useSolverWido.tsx

```diff
@@ -90,16 +92,43 @@ export function useSolverWido(): TSolverContext {
 	** call getQuote to get the current quote for the provided request.current.
 	**********************************************************************************************/
 	const init = useCallback(async (_request: TInitSolverArgs, shouldLogError?: boolean): Promise<TNormalizedBN> => {
-		if (isSolverDisabled[Solver.WIDO] || !_request.inputToken.solveVia?.includes(Solver.WIDO)) {
+		/******************************************************************************************
```

Comment:
As we are repeating this logic for all solvers, I'd move this to a common place and cleanup individual solvers

Reply:
Let's sleep on this, I think testing this code is a bit higher priority... so that we can then safely refactor. TBD